### DISCUSSION TITLE
What mods/types of mod is recommended to use in combo?

### DISCUSSION THREAD

**Discussion by hpf3** [fnuecke/oc2#88] [Q&A]
I am wondering what mods (if any) should I use for trying oc2? For a simple example I remember that oc1, while having a method to generate energy, it was a better experience using another mods generator (and less entity collision checks)

**Comment by CoolCat467** [comment]
I've never played with it myself yet, but I would assume something like mekanism might work well for energy production.

**Comment by fnuecke** [comment]
Anything that's compatible with the Forge Energy capability should work.
